src/process.c

```diff
@@ -463,59 +466,450 @@ cc_run_hook(struct oci_cfg_hook* hook, const gchar* state,
 	return result;
 }
 
+
+/*!
+ * Obtain the network configuration by querying the network namespace.
+ *
+ * \param[in,out] config \ref cc_oci_config.
+ *
+ * \return \c true on success, else \c false.
+ */
+static gboolean
+cc_oci_vm_netcfg_get (struct cc_oci_config *config)
+{
+	if ((! config)) {
+		return false;
+	}
+
+	/* TODO: We need to support multiple networks */
+	if ( !cc_oci_network_discover ("eth0", config)) {
```

Comment:
@sameo Agree. This will be replaced by code that scans the network namespace for all valid network interfaces and sets them up one by one. Today I am relying on the fact that docker libnetwork plugins all create interafaces in the sequence eth0, eth1,....

I will at least comment and list the future implementation here.